src/pages/results.tsx

```diff
@@ -125,9 +133,14 @@ const ResultsPage = ({ patient, user }: ResultsPageProps): ReactElement => {
             sx={{ overflowY: 'auto' }}
           >
             <ResultsHeader isOpen={open} toggleDrawer={toggleDrawer} toggleMobileDrawer={toggleMobileDrawer} />
-
             <MainContent elevation={0} sx={{ flex: '1 1 auto', overflowY: 'auto', p: 3 }} square>
-              <Results data={data} />
+              {(isIdle || isLoading) && (
+                <Stack alignItems="center" direction="column" justifyContent="center" sx={{ height: '100%' }}>
```

Comment:
Because `Stack` is a CSS utility component (like `Box`), you can add css properties directly as attributes. So, you can replace `sx={{ height: '100%' }}` with `height="100%"`.